cvat/apps/events/views.py

```diff
@@ -0,0 +1,151 @@
+        methods=['POST'],
+        description='Sends logs to the Clickhouse if it is connected',
+        request=ClientEventsSerializer(),
+        responses={
+            '201': ClientEventsSerializer(),
+        })
+    @extend_schema(summary='Method returns csv log file ',
+        methods=['GET'],
+        description='Recieve logs from the server',
+        parameters=[
+            OpenApiParameter('org', location=OpenApiParameter.QUERY, type=OpenApiTypes.INT, required=False,
+                description="Filter events by organization ID"),
+            OpenApiParameter('project', location=OpenApiParameter.QUERY, type=OpenApiTypes.INT, required=False,
+                description="Filter events by project ID"),
+            OpenApiParameter('task', location=OpenApiParameter.QUERY, type=OpenApiTypes.INT, required=False,
+                description="Filter events by task ID"),
+            OpenApiParameter('job', location=OpenApiParameter.QUERY, type=OpenApiTypes.INT, required=False,
+                description="Filter events by job ID"),
+            OpenApiParameter('user', location=OpenApiParameter.QUERY, type=OpenApiTypes.INT, required=False,
+                description="Filter events by user ID"),
+            OpenApiParameter('from', location=OpenApiParameter.QUERY, type=OpenApiTypes.DATETIME, required=False,
+                description="Filter events after the datetime"),
+            OpenApiParameter('to', location=OpenApiParameter.QUERY, type=OpenApiTypes.DATETIME, required=False,
+                description="Filter events before the datetime"),
+            OpenApiParameter('filename', description='Desired output file name',
+                location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=False),
+            OpenApiParameter('action', location=OpenApiParameter.QUERY,
+                description='Used to start downloading process after annotation file had been created',
+                type=OpenApiTypes.STR, required=False, enum=['download']),
+            OpenApiParameter('query-id', location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=False,
+                description="ID of query request that need to check or download"),
+        ],
+        responses={
+            '200': OpenApiResponse(description='Download of file started'),
+            '201': OpenApiResponse(description='CSV log file is ready for downloading'),
+            '202': OpenApiResponse(description='Creating a CSV log file has been started'),
+        })
+    @action(detail=False, methods=['GET', 'POST'])
+    def events(request):
+        if request.method == 'POST':
+            serializer = ClientEventsSerializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
+
+            org = request.iam_context['organization']
+            org_id = org.id if org else None
+
+            send_time = datetime_parser.isoparse(serializer.data["send_timestamp"])
```

Comment:
Can we adjust timestamps inside the serializer? What are pros and cons? I believe it is a part of de-serialization logic.